app/src/main/java/com/codesoom/assignment/exception/TaskNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment.exception;
+
+public class TaskNotFoundException extends RuntimeException {
+    public TaskNotFoundException() {
+    }
+
+    public TaskNotFoundException(String message) {
+        super(message);
+    }
+
+    public TaskNotFoundException(Throwable cause) {
+        super(cause);
+    }
+
+    public TaskNotFoundException(String message, Throwable cause) {
+        super(message, cause);
+    }
+}
```

Comment:
아직 필요하지 않은 생성자를 잔뜩 미리 만들어두지 않는 것이 좋습니다. 지금 딱 필요한 것만 정의해보세요.
```suggestion
public class TaskNotFoundException extends RuntimeException {
    public TaskNotFoundException() {
    }

    public TaskNotFoundException(String message) {
        super(message);
    }
}
```